tide_webform.module

```diff
@@ -322,6 +322,13 @@ function tide_webform_webform_submission_presave(WebformSubmission $webform_subm
         $webform_submission->setData($submission_data);
       }
     }
+    // Server side validation from API submission.
```

Comment:
Can we add a comment here explaining this is a workaround while the contrib issue is addressed?

Reply:
done